root/statistics/Index.js

```diff
@@ -0,0 +1,1078 @@
+/*
+ * @flow
+ * Copyright (C) 2018 Shamroy Pellew
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+import {range} from 'lodash';
+
+import manifest from '../static/manifest';
+import {addColon} from '../static/scripts/common/i18n';
+import {lp_attributes} from '../static/scripts/common/i18n/attributes';
+import {l_statistics, ln_statistics, lp_statistics} from '../static/scripts/common/i18n/statistics';
```

Comment:
These should be `l_statistics as l, ln_statistics as ln, lp_statistics as lp`. This is only needed for the statistics domain - attributes for example are extracted from the database, not source files, so `lp_attributes` is fine.